frontend/lib/api/form/pollingstation/usePoliticalGroups.ts

```diff
@@ -46,7 +46,7 @@ export function usePoliticalGroup(
   }, [registerCurrentForm, getValues, political_group_number, getIgnoreWarnings]);
 
   return {
-    saving,
+    status: status,
```

Comment:
Why `status: status` here and only `status` in the other `use... .ts` files?

Reply:
Good catch, this was left over from refactoring (renaming `saving` to `status`). Fixed in cf325c7fded378f2346f2de52e37e7f59fab0427.